ng/src/web/pages/portlists/portrangedialog.js

```diff
@@ -37,21 +37,18 @@ import TextField from '../../components/form/textfield.js';
 
 import Layout from '../../components/layout/layout.js';
 
-const DEFAULTS = {
-  port_type: 'tcp',
-};
-
 const PortRangeDialog = ({
   port_list,
+  port_type = 'tcp',
   title = _('New Port Range'),
   visible,
   onClose,
   onSave,
 }) => {
 
   const data = {
-    ...DEFAULTS,
     ...port_list,
```

Comment:
Same as in PortListDialog. The properties used from port_list should be explicitly defined.